iroh/src/util/fs.rs

```diff
@@ -225,10 +226,48 @@ fn path_content_info0(path: impl AsRef<Path>) -> anyhow::Result<PathContent> {
     Ok(PathContent { size, files })
 }
 
+/// Helper function that translates a path on a filesystem to a key in a document.
+pub fn path_to_key(path: impl AsRef<Path>) -> anyhow::Result<Bytes> {
+    let buf = path
+        .as_ref()
+        .to_str()
+        .map(|p| p.as_bytes())
+        .ok_or(anyhow::anyhow!("could not convert path to bytes"))?;
+    let mut bytes = BytesMut::from(buf);
+    bytes.put(&b"\0"[..]);
+    Ok(bytes.freeze())
+}
+
+/// Helper function that translates a key that was derived from a path on a filesystem back to that
+/// path.
+pub fn key_to_path(key: impl AsRef<[u8]>) -> anyhow::Result<PathBuf> {
```

Comment:
For both these functions users will usually want to strip/apply a path prefix because they likely wouldn't want or should treat doc keys as *absolute* paths. So the utility of these functions in this way is questionable to me now that I see them. Or there should be a way to pass in a path prefix to add/strip I think.

Reply:
great, thank you. I wrote a version of that (adding and stripping prefixes) in the `iroh import` and `iroh export` code paths already, so we can now just move those to the util crate 🙌